### DISCUSSION TITLE
How do I delete a project with no tasks assigned to it?

### DISCUSSION THREAD

**Discussion by sdondley** [GothenburgBitFactory/taskwarrior#2758] [Q&A] [answered]
I have TW set up to show all projects in tab completion. However, old ones with typos in the name are showing up and I'd like to delete them. How can I do this?

**Comment by djmitche** [comment, marked as answer]
I suspect that the tab completion is using `task projects` to get the list of projects.  In that case, it is finding those incorrect project names in completed (or maybe deleted?) projects.  Try deleting and purging the tasks with the incorrect projects, and see if that helps.

**Comment by sdondley** [comment]
Thanks. I ended up figuring it out. Long story short is I had to modify the completed tasks over to correctly spelled project name.

`task +COMPLETE project:bad m project:good`

Feature request: make this more intuitive. I think I've had to google it like 5 times over the years.

**Reply by tbabej** [reply]
Slight remark: If you don't have any context active, `task project:bad mod project:good` should affect all the tasks, including the completed ones :slightly_smiling_face:

**Reply by sdondley** [reply]
Yeah that was exactly one of my problems. I had a context turned on and was running into issues.
